modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java

```diff
@@ -151,12 +151,12 @@ protected void waitUntilContainerStarted() {
                     Thread.sleep(100L);
                 }
             }
-        } catch (InterruptedException e) {
-            Thread.currentThread().interrupt();
-            throw new ContainerLaunchException("Container startup wait was interrupted", e);
         }
 
-        logger().info("Container is started (JDBC URL: {})", JdbcDatabaseContainer.this.getJdbcUrl());
+        throw new IllegalStateException(
+            String.format("Container is started, but cannot be accessed by (JDBC URL: %s), please check container logs",
+                JdbcDatabaseContainer.this.getJdbcUrl())
```

Comment:
Super tiny nit, but I think this can be reduced to:
```suggestion
                this.getJdbcUrl())
```

Other than that though, and an unfortunate merge conflict that I don't have permission to resolve, I think this PR is fine.

Reply:
I updated that `JdbcDatabaseContainer.this.getJdbcUrl())`, also resolved the conflict. Thanks for the review.